SimonImanigirimpuhwe/my-brand-website
# #8 ft(read blog page): add article full page view

#### What does this PR do?

- Enable full view display of selected post

#### Description of Task to be completed?

- Activate all posts to read mode

- Add navigation links to the page

- Add social media links

- Connect blog page to post read page

- Activate interaction menu


#### How should this be manually tested?

- Navigate to blog page, click to any article you wish you will be directed to full view page of that article

- On article page on left corner in interaction menu click on `comment icon` you will see a comment form displayed 

#### Any background context you want to provide?

- N/A

#### What are the relevant Trello stories?

- [#8](https://trello.com/c/h8hefbUI/8-article-full-view-page)

#### Screenshots (if appropriate)
![blog-full-view-screenshoot](https://user-images.githubusercontent.com/54619678/93715359-0e672500-fb69-11ea-9d2d-47a9c69539c4.png)

#### Questions:

- N/A
